docs/guides/using-store-outside-react.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: Using store outside React
+description: Using store outside React
+nav: 4.98
+keywords: state, outside, react
+published: false
+---
+
+Jotai's state resides in React, but sometimes it would be nice
+to interact with the world outside React.
+
+## createStore
+
+[`createStore`](../core/store) provides a store interface that can be used to store your atoms. Using the store, you can access and mutate the state of your stored atoms from outside React.
+
+
+```jsx
+import { atom, useAtomValue, createStore, Provider } from "jotai";
+
+const timeAtom = atom(0);
+const store = createStore();
+
+store.set(timeAtom, store.get(timeAtom) + 1); // Update atom's value
```

Comment:
I think it's better to show function update in this case:
```suggestion
store.set(timeAtom, (prev) => prev + 1); // Update atom's value
```

Reply:
Updates the CodeSandbox too

<img width="784" alt="image" src="https://github.com/pmndrs/jotai/assets/60546840/1bf3807c-8fdc-4fc2-b2ed-3ae73cfe8f56">